source/containers.tex

```diff
@@ -3560,6 +3566,73 @@
 Member function \tcode{swap()} shall have a
 non-throwing exception specification.
 
+\rSec3[array.creation]{Array creation functions}
+
+\indexlibrary{\idxcode{array}}%
+\indextext{\idxcode{array}!creation}%
+
+\indexlibrarymember{array}{to_array}%
+\begin{itemdecl}
+template<class T, size_t N>
+  constexpr array<remove_cv_t<T>, N> to_array(T (&a)[N]);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\mandates
+\tcode{is_array_v<T>} is \tcode{false} and
+\tcode{is_constructible_v<T, T\&>} is \tcode{true}.
+
+\pnum
+\expects
+\tcode{T} meets the \oldconcept{CopyConstructible} requirements.
+
+\pnum
+\returns \tcode{\{\{ a[0], $\dotsc$, a[N - 1] \}\}}.
+\end{itemdescr}
+
+\indexlibrarymember{array}{to_array}%
+\begin{itemdecl}
+template<class T, size_t N>
+  constexpr array<remove_cv_t<T>, N> to_array(T (&&a)[N]);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\mandates
+\tcode{is_array_v<T>} is \tcode{false} and
+\tcode{is_move_constructible_v<T, T\&>} is \tcode{true}.
```

Comment:
Should this be `is_move_constructible_v<T>` or `is_constructible_v<T, T&&>` instead of `is_move_constructible_v<T, T&>`?

Reply:
Thanks - should have been is_move_constructible_v<T>.